github_connector/models/res_partner.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models, fields
+from openerp.tools.translate import _
+
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = ['res.partner', 'abstract.github.model']
+
+    _github_type = 'user'
+    _github_login_field = 'login'
+    _need_individual_call = True
+    _field_list_prevent_overwrite = ['name', 'website', 'email', 'image']
+
+    # Column Section
+    is_bot_account = fields.Boolean(
+        string='Is Bot Github Account', help="Check this box if this"
+        "account is a bot, or something similar.")
+
+    github_team_ids = fields.Many2many(
+        string='Teams', comodel_name='github.team.partner',
+        inverse_name='partner_id', readonly=True)
+
+    github_team_qty = fields.Integer(
+        string='Teams Quantity', compute='_compute_github_team_qty',
+        store=True)
+
+    organization_ids = fields.Many2many(
+        string='Organizations', comodel_name='github.organization',
+        relation='github_organization_partner_rel', column1='partner_id',
+        column2='organization_id', readonly=True)
+
+    organization_qty = fields.Integer(
+        string='Organizations Quantity', compute='_compute_organization_qty',
+        store=True)
+
+    # Constraints Section
+    _sql_constraints = [
+        (
+            'github_login_uniq', 'unique(github_login)',
+            "Two partners with the same Github Login ?"
```

Comment:
Hehe, let's improve this, will you? :wink:
"Two different partners cannot have the same Github Login"